test/msw-api/setup-server/scenarios/concurrency/api.ts

```diff
@@ -0,0 +1,12 @@
+import fetch from 'node-fetch'
+
+export const makeRequest = async (endpoint: string) => {
+  const res = await fetch(endpoint, {
+    method: 'GET',
+    headers: {
+      'Content-Type': 'application/json',
```

Comment:
Specifying `Content-Type` for `GET` requests does nothing as GET requests cannot have body (content).